recipe/meta.yaml

```diff
@@ -1,35 +1,64 @@
-{% set name = "glad2" %}
 {% set version = "2.0.4" %}
 
 package:
-  name: {{ name }}
+  name: glad2
   version: {{ version }}
 
 source:
   url: https://github.com/Dav1dde/glad/archive/refs/tags/v{{ version }}.tar.gz
   sha256: 02629644c242dcc27c58222bd2c001d5e2f3765dbbcfd796542308bddebab401
+  patches:
+    - 0001-cmake.patch
 
 build:
-  number: 0
-  entry_points:
-    - glad = glad.__main__:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-
-requirements:
-  host:
-    - pip
-    - python >=3.6
-    - setuptools_scm
-  run:
-    - python >=3.6
-    - jinja2
-
-test:
-  imports:
-    - glad
-  commands:
-    - glad --help
+  number: 1
+
+outputs:
+  - name: glad2
+    build:
+      noarch: python
+      entry_points:
+        - glad = glad.__main__:main
+      script: python -m pip install . --no-deps -vv
+
+    requirements:
+      host:
+        - pip
+        - python >=3.8
+      run:
+        - python >=3.8
+        - jinja2
+      run_constrained:
+        - {{ pin_subpackage('glad2-cmake', exact=True) }}
+
+    test:
+      imports:
+        - glad
+      commands:
+        - glad --help
+
+  - name: glad2-cmake
+    build:
+      noarch: python
+
+    script: install_cmake.sh
+
+    files:
+      - lib/cmake/Glad/GladConfig.cmake
+      - Library/lib/cmake/Glad/GladConfig.cmake
```

Comment:
@traversaro 

Sorry for asking.
I'm still not sure.
How good an idea is it to use two files to support noarch: python?
Can I make the decision myself?

I haven't seen any examples of this in conda-forge.

Reply:
As far as I know, all CMake libraries (like anything that needs to install stuff in `${CONDA_PREFIX}` or `${CONDA_PREFIX}\Library` depending on the operating system, such as header only C++ libraries) are not noarch packages, see for example https://github.com/conda-forge/ycm-cmake-modules-feedstock/blob/main/recipe/meta.yaml that is a pure CMake library. I personally do not like installing stuff outside `$CONDA_PREFIX\Library` on Windows, or inside  `${CONDA_PREFIX}\Library` not on Windows. Do you have any specific reason why you want `glad2-cmake` to be noarch?

Reply:
See https://github.com/conda/conda/issues/6978 for what it would take to actually have noarch packages for header only or CMake only libraries. However, it does not seem that anyone actually ever worked on that.

Reply:
There is no specific reason for this.
But it is a bit tricky.
I don't want to break noarch of main package.
You can take a look at the previous [iteration](https://github.com/conda-forge/glad-feedstock/blob/795c35b1ac2d66d93f944093a5ad0e8787107d84/recipe/meta.yaml).
And most importantly, 4 different noarch packages are assembled for main package.
Looks like I just need to use skip.
Will try.

Reply:
What is the problem.with previous iteration?

Reply:
There are four different packages for glad2 being assembled not one.

Reply:
You can look at 
[glad2-2.0.4-pyha770c72_1.conda](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=867492&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=1438)
[glad2-2.0.4-pyh57928b3_1.conda](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=867492&view=logs&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=6119ccbe-9301-594f-7c27-f792b80a7fcc&l=1501)